rules-reviewed/camel3/camel2/xml-removed-components.windup.xml

```diff
@@ -86,6 +86,20 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="xml-removed-components-00005">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-restlet" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-restlet` artifact has been removed" effort="7" category-id="mandatory" >
```

Comment:
tbh, it feels like 7 sp of effort could be a lot ? it would mean "The migration requires a complete rearchitecture of the component or subsystem. "
You can check the sp table here : 
https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/4.1/html-single/rules_development_guide/index#how_story_points_are_estimated_in_rules

Reply:
Hi @jonathanvila thanks for reviewing! I've changed it to 3